tests/test_live.py

```diff
@@ -0,0 +1,265 @@
+import io
+import os
+import random
+import sys
+import time
+from typing import List
+
+import pytest
+from rich.console import Console
+from rich.live import Live
+from rich.table import Table
+from rich.text import Text
+
+from tests.util import get_capture_text, set_capture_text
+
+
+def create_capture_console(*, width: int = 60, height: int = 80) -> Console:
```

Comment:
I'd prefer to avoid external helper functions for tests. I think its best to have the entire test located in a single test function (other than the code being tested obviously).